pkg/nsx/services/common/builder.go

```diff
@@ -54,3 +55,13 @@ func ParseVPCResourcePath(nsxResourcePath string) (VPCResourceInfo, error) {
 	info.ParentID = layers[size-3]
 	return info, nil
 }
+
+// parse org id and project id from nsxProject path
+// example /orgs/default/projects/nsx_operator_e2e_test
+func NSXProjectPathToId(path string) (string, string, error) {
+	parts := strings.Split(path, "/")
+	if len(parts) < 4 {
```

Comment:
I guess "/orgs/default/projects" can pass this validation, but it is supposed not valid?

Reply:
Updated 4 to 5, thanks!